google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -193,6 +237,8 @@ int main(int argc, char* argv[]) try {
       {"get-bucket-metadata", &GetBucketMetadata},
       {"delete-bucket", &DeleteBucket},
       {"change-default-storage-class", &ChangeDefaultStorageClass},
+      {"patch-bucket", &PatchBucket},
```

Comment:
This isn't a generic Bucket patch operation. How about`patch-bucket-storage-class`?

Reply:
Done. But note that many of the other examples are "not a generic X operation" either. We have `create-bucket`, but it creates with defaults, and `insert-object` doesn't do anything too fancy, these are examples, not a replacement for `gsutil` (thankfully).

Happy to revert if that changes your mind, or to leave the new changes if it does not change your mind.